packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -982,6 +982,28 @@ IWorker engine = WorkerFactory.CreateWorker(BackendType.GPUCompute, model);
 `,
 ];
 
+export const sana = (model: ModelData): string[] =>	[
+		`
+# Install from https://github.com/NVlabs/Sana
+
+# Load the model and infer image from text
```

Comment:
```suggestion
# Load the model and infer image from text
```

No need to link github repo in snippet. A button will be shown just above the snippet for the user to click on. See example for seed-story repo:
![image](https://github.com/user-attachments/assets/9f206cb6-e8bc-4136-b9b5-cdd5689df347)